pom.xml

```diff
@@ -14,7 +14,8 @@
         <failOnMissingWebXml>false</failOnMissingWebXml>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <jackson.version>2.9.6</jackson.version>
-        <openshift.fabric8.version>4.0.3</openshift.fabric8.version>
+        <openshift.fabric8.version>4.0.4</openshift.fabric8.version>
```

Comment:
is that the fmp ?

Reply:
No, it's the fabric8 openshift client :)